packages/thirdweb/src/wallets/aws-kms.test.ts

```diff
@@ -0,0 +1,143 @@
+import { beforeAll, expect, test } from "vitest";
+import { TEST_CLIENT } from "~test/test-clients.js";
+import { typedData } from "~test/typed-data.js";
+import { ANVIL_CHAIN } from "../../test/src/chains.js";
+import { sendTransaction } from "../exports/thirdweb.js";
+import { prepareTransaction } from "../transaction/prepare-transaction.js";
+import { getAwsKmsAccount } from "./aws-kms.js";
+import { getWalletBalance } from "./utils/getWalletBalance.js";
+
+import {
+  http,
+  createTestClient,
+  parseUnits,
+  verifyMessage,
+  verifyTypedData,
+} from "viem";
+import { TEST_AWS_KMS_CONFIG } from "~test/test-aws-kms-config.js";
+import { toWei } from "../utils/units.js";
+
+let account: Awaited<ReturnType<typeof getAwsKmsAccount>>;
+
+const anvilTestClient = createTestClient({
+  mode: "anvil",
+  transport: http(ANVIL_CHAIN.rpc),
+});
+
+beforeAll(async () => {
+  account = await getAwsKmsAccount({
```

Comment:
So this will always hit a live KMS account? And is there any way to keep that address constant?

Reply:
Same KMS keyId would get you the same address. One way to enforce this on a test would be to also expose `TEST_AWS_KMS_EXPECTED_ADDRESS` and verify it matches.